src/Layer/GeometryLayer.js

```diff
@@ -37,6 +37,10 @@ function traverse(obj, callback) {
  * @property {boolean} isGeometryLayer - Used to checkout whether this layer is
  * a GeometryLayer. Default is true. You should not change this, as it is used
  * internally for optimisation.
+ * @property {number} [zoom.max=Infinity] - this is the maximum zoom beyond which it'll be hidden.
```

Comment:
Why can't someone set the maximum value ?

Reply:
Yes!
one could imagine that the layer becomes invisible when this zoom level is reached.

Reply:
For **`GeometryLayer`**, the properties **`zoom.max`** and **`zoom.min`** are used if the layer is attached to **`TileLayer`**. This is tile level that determines the **`GeometryLayer`** displaying. That is, if **`GeometryLayer.zoom.min == 6`**  and a **tile with level of 6** is displayed, then the features geometry, inside of this tile extent are displayed.

This could make sense if the features were fetched/parsed for each level but for the moment the data would be the same.

Finally,  I keep **`zoom.max`** at **`Infinity`**, because it's complex to stop displaying when all intermediates levels aren't loaded, (like raster).

Reply:
Sounds good to me. Maybe specify in this documentation that there is not a maximum level after it is hidden.

Reply:
doc updated